contentcuration/contentcuration/serializers.py

```diff
@@ -50,7 +50,7 @@ class Meta:
         fields = ("__all__")
 
 
-class FormatPresetSerializer(serializers.ModelSerializer):
+class FormatPresetSerializer(serializers.PrimaryKeyRelatedField):
```

Comment:
Why do we need this change?

Reply:
Richard suggested that `ModelSerializer` was responsible for DRF attempting to create a new `FormatPreset`, and that we instead use a `PrimaryKeyRelatedField` in order to look up the existing `FormatPreset`.